proto/cosmos/auth/query.proto

```diff
@@ -11,10 +12,14 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/auth/types";
 // Query creates service with Account and Parameters as rpc
 service Query{
     // Account returns account details based on address
-    rpc Account (QueryAccountRequest) returns (QueryAccountResponse) {}
+    rpc Account (QueryAccountRequest) returns (QueryAccountResponse) {
+      option (google.api.http).get = "/cosmos/auth/v1beta/account/{address}";
```

Comment:
Just a small note: the current proto package version in #6822 is `v1beta1`. That might still change, but these annotations should match what's there.